ui/k-toolbar/main.kytos

```diff
@@ -78,9 +98,41 @@
                 After trying to create a MW, show a modal if end_time is empty, otherwise creates it.
             */
             create_window: function() {
-                if (!this.end) {this.show_post_modal = true}
+                if (!this.end && this.time_date) {this.show_post_modal = true}
                 else {this.post_window()}
             },
+            /*
+                Get the time zone in +HHMM format
+            */
+            get_time_zone: function() {
+                var zone_min = new Date().getTimezoneOffset()
+                let sign = zone_min > 0 ? "-": "+"
```

Comment:
```suggestion
                let sign = zone_min >= 0 ? "+": "-"
```

Aren't the signs swapped? Could you double check?

Reply:
Delete it